models/spring-ai-mistral-ai/src/test/java/org/springframework/ai/mistralai/MistralAiRetryTests.java

```diff
@@ -135,6 +136,7 @@ public void mistralAiChatNonTransientError() {
 	}
 
 	@Test
+	@Disabled("Currently stream() does not implement retry")
```

Comment:
It will be implemented in https://github.com/spring-projects/spring-ai/issues/1193